record_tx_strategy_external_incoming_tx.go

```diff
@@ -9,9 +9,11 @@ import (
 )
 
 type externalIncomingTx struct {
-	Hex                  string
+	BtTx                 *bt.Tx
 	broadcastNow         bool // e.g. BEEF must be broadcasted now
 	allowBroadcastErrors bool // only BEEF cannot allow for broadcast errors
+
+	txId string
```

Comment:
Minor - lint error:
`var-naming: struct field txId should be txID (revive)go-golangci-lint`

Reply:
Done